rb/spec/integration/selenium/webdriver/manager_spec.rb

```diff
@@ -242,6 +242,11 @@ module WebDriver
           expect(driver.manage.all_cookies.find { |c| c[:name] == 'foo' }).to be_nil
         end
 
+        it 'throws error when cookie name is empty string' do
+          expect { driver.manage.delete_cookie('') }
+            .to raise_exception(ArgumentError)
```

Comment:
@pallavigitwork this test is failing due to **expected ArgumentError but nothing was raised**  you need to pass nil or an empty hash to make this test work

Reply:
ok thank you @aguspe . I will try to fix this by this thursday. i will work on the feedback you gave. thank you for your time.

Reply:
made changes. requesting review,  @aguspe